### PR TITLE
Add a note about scaling above 100,000 registered users, reference loadtest framework

### PR DESCRIPTION
**Pull request by jasonblais**
Based on feedback from @wget

### CODE REVIEW

File: source/deployment/scaling.rst [hunk 1]
```diff
@@ -68,6 +68,8 @@ Hosting Recommendation for 100,000+ users
 
 The following matrix presents key features for a successful multi-region Mattermost implementation that scales to 100,000 users with support for high availability and geographically based traffic routing in on premises, AWS, and Azure deployments.
 
+To scale to 100,000 users and above, it is recommended to use the Mattermost open source load testing framework to simulate usage of your system at full scale: https://github.com/mattermost/mattermost-load-test.
```
Comment (cwarnermm) [inline source/deployment/scaling.rst:71]:
```suggestion
To scale to 100,000 users and above, we recommended using `the Mattermost open source load testing framework <https://github.com/mattermost/mattermost-load-test.>`__ to simulate usage of your system at full scale.
```

**Review by cwarnermm — COMMENTED** [review]
Suggestion proposed inline.

**Review by wiersgallak — APPROVED** [review]

**Review by cwarnermm — APPROVED** [review]
Thanks!

File: source/deployment/scaling.rst [hunk 2]
```diff
@@ -68,6 +68,8 @@ Hosting Recommendation for 100,000+ users
 
 The following matrix presents key features for a successful multi-region Mattermost implementation that scales to 100,000 users with support for high availability and geographically based traffic routing in on premises, AWS, and Azure deployments.
 
+To scale to 100,000 users and above, we recommended using `the Mattermost open source load testing framework <https://github.com/mattermost/mattermost-load-test.>`__ to simulate usage of your system at full scale.
```
Comment (amyblais) [inline source/deployment/scaling.rst:71]: @jasonblais @cwarnermm The link here is broken.

Comment (cwarnermm) [reply source/deployment/scaling.rst:71]: Thanks @amyblais - Fixing via https://github.com/mattermost/docs/pull/4560

**Comment by wget** [comment]
LGTM. We can merge (if we merge #4560 as well) =P

**Pull request merged**